src/PopEngine/index.ts

```diff
@@ -197,14 +211,14 @@ export class PopEngine {
     return false;
   }
 
-  // private _isPopAlreadyOpen(targetElement: Element): boolean {
-  //   let groupId = targetElement.getAttribute('popgun-group');
-  //   if (this.getPopFromGroupId(groupId)) {
-  //     return ((this.getPopFromGroupId(groupId).state === PopStateType.SHOWING) &&
-  //       (this.getPopFromGroupId(groupId).targetEl === targetElement));
-  //   }
-  //   return false;
-  // }
+  public isPopAlreadyOpen(targetElement: Element): boolean {
```

Comment:
In Typescript, all public member functions must appear above the private member functions (none of it matters because typescript all gets compiled the javascript and everything is a public method, but let's move all the new public methods above the private methods).